model/license.go

```diff
@@ -259,6 +261,11 @@ func (f *Features) SetDefaults() {
 	if f.RemoteClusterService == nil {
 		f.RemoteClusterService = NewBool(*f.FutureFeatures)
 	}
+
+	if f.CustomGroups == nil {
+		// f.CustomGroups = NewBool(false) // TODO: change to this
+		f.CustomGroups = NewBool(*f.FutureFeatures) // TODO: remove this, it's just for testing
```

Comment:
This must be updated prior to merge. It's temporarily like this to avoid needing to create a new E20 license file, for now.